aten/src/ATen/core/dispatch/OperatorEntry.cpp

```diff
@@ -137,5 +139,54 @@ void OperatorEntry::updateDispatchTable_(c10::optional<DispatchKey> dispatch_key
   }
 }
 
+void OperatorEntry::checkInvariants() const {
+  if (schema_) {
```

Comment:
> Did you evaluate if we can delete the name_ member and just use dispatchTable_.operatorName() everywhere instead?

I suppose this is possible, although that kind of seems like a smell to me; logically, it's the OperatorEntry that has a name, and the DispatchTable is just an internal implementation detail of the actual "fast path" table. So a better refactor would be to see how to eliminate `dispatchTable_.operatorName()` (but I can't conveniently do that because then I can't print good error messages in dispatch table!) So where this actually makes me lean is that DispatchTable should be folded into OperatorEntry. What do you think about this?